app/src/main/java/at/irfc/app/ui/program/ProgramScreen.kt

```diff
@@ -55,14 +63,82 @@ fun ProgramScreen(
+                            .padding(bottom = 16.dp)
+                            .fillMaxWidth()
+                    ) {
+                        Row(
+                            modifier = Modifier
+                                .clickable {
+                                    navController.navigate(
+                                        ProgramDetailScreenDestination(event.id)
+                                    )
+                                }
+                                .fillMaxWidth()
+                        ) {
+// Box for the image
+                            Box(
+                                modifier = Modifier
+                                    .width(150.dp)
+                                    .aspectRatio(1f)
+                                    .clip(RoundedCornerShape(20.dp))
+                                    .background(Color.White)
+                            ) {
+                                AsyncImage(
+                                    model = event.image.path,
+                                    contentDescription = event.image.title,
+                                    contentScale = ContentScale.FillBounds,
+                                    modifier = Modifier
+                                        .fillMaxSize()
+                                        .clip(RoundedCornerShape(20.dp))
+
+                                )
+                            }
+
+// Text on the right side of the image
+                            Column(
+                                modifier = Modifier
+                                    .padding(
+                                        start = 16.dp,
+                                        top = 30.dp,
+                                        end = 30.dp,
+                                        bottom = 30.dp
+                                    )
+                                    .fillMaxWidth()
+                            ) {
+                                Text(
+                                    text = event.title,
+                                    style = MaterialTheme.typography.bodyLarge.copy(
+                                        fontWeight = FontWeight.Bold
+                                    ),
+                                    maxLines = 2,
+                                    overflow = TextOverflow.Ellipsis
+                                )
+
+                                Spacer(modifier = Modifier.height(5.dp))
+
+                                Text(
+                                    text = "${SimpleDateFormat("HH:mm").format(event.startDate)}" +
+                                        " - " + SimpleDateFormat("HH:mm").format(event.endDate),
+                                    style = MaterialTheme.typography.bodyMedium.copy(
+                                        fontWeight = FontWeight.Bold
+                                    )
+                                )
```

Comment:
I think this time of the event should not be in the column

see design: <img width="566" alt="Bildschirmfoto 2023-05-01 um 16 51 34" src="https://user-images.githubusercontent.com/73596295/235471218-a07af84a-f569-4308-a18b-07318923136c.png">